tests/baselines/reference/unconstrainedTypeComparison.errors.txt

```diff
@@ -0,0 +1,109 @@
+unconstrainedTypeComparison.ts(2,12): error TS18046: 'a' is of type 'unknown'.
+unconstrainedTypeComparison.ts(2,16): error TS18046: 'b' is of type 'unknown'.
+unconstrainedTypeComparison.ts(6,12): error TS18049: 'a' is possibly 'null' or 'undefined'.
+unconstrainedTypeComparison.ts(6,16): error TS18049: 'b' is possibly 'null' or 'undefined'.
+unconstrainedTypeComparison.ts(10,12): error TS18046: 'a' is of type 'unknown'.
+unconstrainedTypeComparison.ts(10,16): error TS18046: 'b' is of type 'unknown'.
+unconstrainedTypeComparison.ts(14,12): error TS18046: 'a' is of type 'unknown'.
+unconstrainedTypeComparison.ts(14,16): error TS18046: 'b' is of type 'unknown'.
+unconstrainedTypeComparison.ts(18,12): error TS18049: 'a' is possibly 'null' or 'undefined'.
+unconstrainedTypeComparison.ts(18,16): error TS18049: 'b' is possibly 'null' or 'undefined'.
+unconstrainedTypeComparison.ts(22,12): error TS18046: 'a' is of type 'unknown'.
+unconstrainedTypeComparison.ts(22,16): error TS18046: 'b' is of type 'unknown'.
+unconstrainedTypeComparison.ts(26,12): error TS18048: 'a' is possibly 'undefined'.
+unconstrainedTypeComparison.ts(26,16): error TS18048: 'b' is possibly 'undefined'.
+unconstrainedTypeComparison.ts(30,12): error TS18047: 'a' is possibly 'null'.
+unconstrainedTypeComparison.ts(30,16): error TS18047: 'b' is possibly 'null'.
+unconstrainedTypeComparison.ts(34,12): error TS18049: 'a' is possibly 'null' or 'undefined'.
+unconstrainedTypeComparison.ts(34,16): error TS18049: 'b' is possibly 'null' or 'undefined'.
+unconstrainedTypeComparison.ts(45,12): error TS18047: 'a' is possibly 'null'.
+unconstrainedTypeComparison.ts(45,16): error TS18048: 'b' is possibly 'undefined'.
+
+
+==== unconstrainedTypeComparison.ts (20 errors) ====
+    function f1<T>(a: T, b: T): boolean {
+        return a > b;
+               ~
+!!! error TS18046: 'a' is of type 'unknown'.

```

Comment:
This error message isn't right, though I'm not sure what. `possilby 'null' or 'undefined'` might be alright, since those are the values we're concerned about entering a comparison? And then it's consistent with a constraint of `{} | null | undefined`.

Reply:
`possibly 'null' or 'undefined'` is the one that comes up with the other PR